python/pyspark/sql/tests/connect/test_connect_basic.py

```diff
@@ -84,6 +84,15 @@ class SparkConnectSQLTestCase(ReusedConnectTestCase, SQLTestUtils, PandasOnSpark
     """Parent test fixture class for all Spark Connect related
     test cases."""
 
+    @classmethod
```

Comment:
I actually played with this for scala tests, and I didn't really notice a difference.